src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -114,13 +117,46 @@ public void setSendToTis(boolean sendToTis) {
     }
 
     /**
-     * @return {@code true} to continuously retry on error
+     * @return the initial interval retry time that will be used as we graduate to larger numbers between retries
      */
-    public boolean isRetryOnError() {
-        return retryOnError;
```

Comment:
Removed this property; retry behavior is now managed by the `ExamResultsTransmitterMessagingConfiguration` class.